fsharp-backend/tests/FuzzTests/ExecutePureFunctions.FuzzTests.fs

```diff
@@ -179,7 +185,8 @@ module Generators =
       if isKnownDifference then
         false
       elif allowedErrors.functionToTest = None then
-        // FSTODO: Add JWT and X509 functions here
+        // FSTODO: Add JWT functions here
```

Comment:
the JWT fns are:
- JWT::signAndEncode_v0, v1
- JWT::signAndEncodeWithHeaders_v0, v1
- JWT::verifyAndExtract_v0, v1

signAndEncode have already been marked as ImpurePreviewable, and I've tested that these work ok against the (static) cert generator.

`verifyAndExtract` is a bit tricky - I believe the fns in Json.FuzzTests.fs are sufficient here?

I recall having some issues fuzztesting `signAndEncodeWithHeaders`, but forget the details now - will follow up with those tomorrow.